core/base/mergeTreeTemporalReductionDecoding/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+ttk_add_base_library(mergeTreeTemporalReductionDecoding
+  SOURCES
+    MergeTreeTemporalReductionDecoding.cpp
+  HEADERS
+    MergeTreeTemporalReductionDecoding.h
+  DEPENDS
+    triangulation
+    ftmTree
```

Comment:
do you need this dep for the `ftmTreeUtils`? (traversal and such)

Reply:
Removed in the next commit, thanks!